_docs/tasks/basic-access-control.md

```diff
@@ -18,54 +18,66 @@ This task shows how to use Istio to control access to a service.
 
 * Deploy the [BookInfo]({{home}}/docs/samples/bookinfo.html) sample application.
 
-* Initialize the application version routing by either first performing the
-  [request routing](./request-routing.html) task or by running the following
-  commands:
-  
+* Initialize the application version routing to direct `reviews` service requests from
+  test user "jason" to version v2 and requests from any other user to v3.
+
   ```bash
-  istioctl create -f samples/apps/bookinfo/route-rule-all-v1.yaml
-  istioctl replace -f samples/apps/bookinfo/route-rule-reviews-v2-v3.yaml
+  istioctl create -f samples/apps/bookinfo/route-rule-reviews-test-v2.yaml
+  istioctl create -f samples/apps/bookinfo/route-rule-reviews-v3.yaml
   ```
-
-* Ensure that you can use [istioctl mixer]({{home}}/docs/reference/commands/istioctl.html#istioctl_mixer) by setting up port forwarding if needed.
+  
+  > Note: if you have conflicting rule that you set in previous tasks,
+    use `istioctl replace` instead of `istioctl create`.
 
 ## Access control using _denials_ 
 
 Using Istio you can control access to a service based on any attributes that are available within Mixer.
 This simple form of access control is based on conditionally denying requests using Mixer selectors.
 
 Consider the [BookInfo]({{home}}/docs/samples/bookinfo.html) sample application where the `ratings` service is accessed by multiple versions
-of the `reviews` service. We would like to cut off access to version `v3` of this service.
+of the `reviews` service. We would like to cut off access to version `v3` of the `reviews` service.
 
-1. Check that versions `v2,v3` of the `reviews` service can access the `ratings` service. 
-   You should see red and black stars alternate when repeatedly visiting 
-   `http://$GATEWAY_URL/productpage` in a browser. 
+1. Point your browser at the Bookinfo `productpage` (http://$GATEWAY_URL/productpage). 
 
-2. Explicitly deny access to version `v3` of the `reviews` service. 
+   If you log in as user "jason", you should see black ratings stars with each review,
+   indicating that the `ratings` service is being called by the "v2" version of the `reviews` service.
+   
+   If you log in as any other user (or logout) you should see red ratings stars with each review,
+   indicating that the `ratings` service is being called by the "v3" version of the `reviews` service.
+
+1. Explicitly deny access to version `v3` of the `reviews` service. 
 
    ```bash
-   istioctl mixer rule create global ratings.default.svc.cluster.local -f deny-reviews.yml
+   istioctl mixer rule create global ratings.default.svc.cluster.local -f deny-reviews.yaml
```

Comment:
I had intended to use/add yaml files in istio/istio (some are there already, some are there, but not exactly correct, some need to be added), but I abandoned that because the smoketest depends on them (Bazel does, even if the code isn't even calling some of them), so given the pain it created to try that (my changes broke the build in several places ... would need to rework the smoketest), I gave up and just left this as Mandar had it (user needs to create the file with copy/paste). That said, I think I'll take your suggested and change it to do the pipe from stdin.